src/views/Settings.vue

```diff
@@ -396,6 +404,9 @@ export default {
       if (isConnected && !this.edgeVersion) {
```

Comment:
@vickywane Sine edgeVersion is synced with the vuex store, I think we should fetch edgeversion info each time a new connection is established. The reason is that the edge device version might have changed due to an automatic edge software update. The UI needs to be able to pick that up. Please think about it and cover with test cases.

Reply:
I have addressed this concern by removing the second `!this.edgeVersion` value from the `if` conditional. 

With this, each time the `isEdgeConnected` method is triggered when an PWA connects after a disconnection from an Edge Device, the edge device's version will be fetched.